src/hooks-usage/useDog.tsx

```diff
@@ -0,0 +1,35 @@
+import Card from "../components/Card"
+import { useDog } from "../hooks/useDog"
+
+const DogCard = () => {
+  const dog = useDog()
+
+  return (
+    <>
+      <div>
+        <button
+          onClick={() => {
+            dog("just a dog")
+          }}
+        >
+          🐈 Say woof
+        </button>
+      </div>
+    </>
```

Comment:
Useless Fragment
(Do we need code quality in this useless repository?)

Reply:
This is by intention. Otherwise it will not similar to useCat.